server/metric/common.go

```diff
@@ -16,17 +16,44 @@
  */
 package metric
 
-import "time"
+import (
+	"github.com/astaxie/beego"
+	"net"
+	"sync"
+	"time"
+)
 
 const (
-	defaultMetricsSize = 100
-	collectInterval    = 5 * time.Second
+	collectInterval  = 5 * time.Second
```

Comment:
i think there are no necessary to make it configurable, because the gather just used internal for some special situations, like calculate the QPS and adjust whether to raise an overload alarm. the interval is not the report metrics period.